packages/rocketchat-lib/server/functions/getFullUserData.js

```diff
@@ -23,13 +23,30 @@ RocketChat.getFullUserData = function({userId, filter, limit}) {
 			services: 1,
 			requirePasswordChange: 1,
 			requirePasswordChangeReason: 1,
-			roles: 1,
-			customFields: 1
+			roles: 1
 		});
 	} else if (limit !== 0) {
 		limit = 1;
 	}
 
+	const metaCustomFields = RocketChat.settings.get('Accounts_CustomFields').trim();
+	let sCustomFields;
+	try {
+		sCustomFields = JSON.parse(metaCustomFields);
+
+		if (sCustomFields) {
+			_.each(sCustomFields, (el, key) => {
```

Comment:
Let's change this to `Object.keys(sCustomFields).forEach((key) => { const item = sCustomFields[key]; })`, as we are slowly trying to lift our reliance on the `_` package.